Lib/glob.py

```diff
@@ -510,14 +510,10 @@ def select_exists(self, path, exists=False):
         """
         if exists:
```

Comment:
We can simply this:

```suggestion
        if exists or self.lexists(path):
```

Reply:
We can, but:

- It makes the comment less clear
- It will be split back up when #117737 lands

Reply:
The comment could be updated:

> Optimization: if this path is already known to exist, e.g. because it was returned from os.scandir(), we skip calling lexists().

And I don't think you need to split it up:

```python
if exists or (dir_fd is None and self.lexists(path)):
```

But if you disagree, we can leave it as it's written right now.

Reply:
But it will indeed have to be split up. This wouln't return when `lexists()` returns `False`:

```none
if exists or (dir_fd is None and self.lexists(path)):
```

Reply:
@erlend-aasland, could you resolve this?